xblockmufi/private/view.js

```diff
@@ -4,8 +4,91 @@ function XblockMufiView(runtime, element) {
     var $ = window.jQuery;
     var $element = $(element);
 
-    // TODO: Put your logic here
+    var handlerUrl = runtime.handlerUrl(element, 'student_submit');
+	var publishUrl = runtime.handlerUrl(element, 'publish_event');
```

Comment:
Actually, there are a number of formatting/indent issues throughout this file.

Can you fix all of these to conform to the style guide?